cli/cmd/component-delete.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/kinvolk/lokomotive/pkg/components"
+	"github.com/kinvolk/lokomotive/pkg/components/util"
+	"github.com/kinvolk/lokomotive/pkg/config"
+)
+
+var componentDeleteCmd = &cobra.Command{
+	Use:   "delete",
+	Short: `Delete a component installed in the cluster.`,
+	Run:   runDelete,
+}
+
+//nolint:gochecknoinits
+func init() {
+	componentCmd.AddCommand(componentDeleteCmd)
+}
+
+func runDelete(cmd *cobra.Command, args []string) {
+	contextLogger := log.WithFields(log.Fields{
+		"command": "lokoctl component delete",
+		"args":    args,
+	})
+
+	lokoCfg, diags := getLokoConfig()
+	if len(diags) > 0 {
+		contextLogger.Fatal(diags)
+	}
+
+	var componentsToDelete []string
```

Comment:
You can do `componentsToDelete := append(componentsToDelete, args...)`, and then `if len(componentsToDelete) <= 0`, so there is no need to use `else`.